### DISCUSSION TITLE
Winpilot 2024.5.5 Adblock

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#199] [Announcements]
Seems like we're all getting tired and annoyed by the new advertising on the Windows 11 UI, right? That's why Winpilot is introducing a new feature/plugin called **Adblock for Windows.** It focuses solely on removing ads from the Windows 11 UI. Special thanks to [elevenforum](https://www.elevenforum.com/t/disable-ads-in-windows-11.8004 ) for the inspiration!

![explorer_ntSeQdkFfY](https://github.com/builtbybel/Winpilot/assets/57478606/7554a919-c941-4a98-bb06-6fc4a5f8c79b)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Winpilot/releases/tag/2024.5.5'>Winpilot 2024.5.5 Adblock</a>.</em>

**Comment by IndrasChild** [comment]
![Screenshot 2024-05-05 184901](https://github.com/builtbybel/Winpilot/assets/169008724/bce2d562-5c38-4177-80a6-4a04c936bfad)


I'm getting this error code whenever I try to run WinPilot, what's going wrong?

**Reply by Belim** [reply]
Edge Browser uninstalled? Perhaps the Evergreen Bootstrapper can help. You can download it here https://developer.microsoft.com/microsoft-edge/webview2/?form=MA13LH#download
